axelrod/strategies/axelrod_first.py

```diff
@@ -189,10 +189,23 @@ class FirstByDowning(Player):
     is assumed to be the behaviour. Interestingly, in future tournaments this
     strategy was revised to not defect on the opening two rounds.
 
-    Note that response to the first round allows us to estimate
+    It is assumed that these first two rounds are used to create initial
+    estimates of
     beta = P(C_o | D_s) and we will use the opening play of the player to
-    estimate alpha = P(C_o | C_s). This is an assumption with no clear
-    indication from the literature.
+    estimate alpha = P(C_o | C_s).
+    Thus we assume that the opponents first play is a response to a cooperation
+    "before the match starts".
+
+    So for example, if the plays are:
+
+    [(D, C), (D, C)]
+
+    Then the opponent's first cooperation counts as a cooperation in response to
+    the non existent cooperation of round 0. The total number of cooperations in
+    response to a cooperation is 1. We need to take in to account that extra
+    phantom cooperation to estimate the probability alpha=P(C|C) as 1 / 1 = 1.
```

Comment:
I have one question, why we don't start with `alpha=P(C|C) = 0.5` as stated above?

Reply:
Because that doesn't necessarily always imply 2 defections as an opening which is one of the "stronger" points made in the various literature. (But I'm guessing here.)

Reply:
I'll investigate more and get back to you (and add to the docstring as well) :+1:

Reply:
6c46483 adds to the docstring on this topic and also no the other points you raised. Let me know what you think.

Reply:
The explanation is really good now! One minor comment would be to change `P(C | C)` to `P(C_o | C_s)` to be consistent.

Reply:
Yup good call.